Frontend/src/components/listUpdateProfileFields.js

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import {ScrollView, View, Text, TextInput, StyleSheet} from 'react-native';
+
+import UpdateProfileField from '../components/updateProfileField';
+
+const ListUpdateProfileField = ({
+  userReducer,
+  setUserEmail,
+  setUserUsername,
+  setUserPassword,
+  setUserPhone,
+  setUserAddress,
+  setUserBirthday,
+  setUserGender,
+  setUserName,
+}) => {
+  return (
+    <ScrollView persistentScrollbar={true}>
+      <UpdateProfileField
+        text="Email"
+        defaultValue={userReducer.email}
+        setData={setUserEmail}
+      />
+      <UpdateProfileField
+        text="Username"
+        defaultValue={userReducer.username}
+        setData={setUserUsername}
+      />
+      <UpdateProfileField
+        text="Password"
+        defaultValue={userReducer.password}
+        setData={setUserPassword}
+        secureTextEntry={true}
+        placeholder="Enter Password"
+      />
+      <UpdateProfileField
+        text="Phone"
+        defaultValue={userReducer.phone}
+        setData={setUserPhone}
+      />
+      <UpdateProfileField
+        text="Address"
+        defaultValue={userReducer.address}
+        setData={setUserAddress}
+      />
+      <UpdateProfileField
+        text="Birthday"
+        defaultValue={userReducer.birthday}
+        setData={setUserBirthday}
+      />
+      <UpdateProfileField
+        text="Gender"
+        defaultValue={userReducer.gender}
+        setData={setUserGender}
+      />
+      <View>
+        <Text style={styles.fieldName}>First Name</Text>
```

Comment:
User first and last name couldn't be an UpdateProfileField? I observed you have a method setUserName that you don't use and instead you set first and last names separate. Maybe creating two methods for setUserFirst/LastName would help